operator/apis/loki/v1/lokistack_types.go

```diff
@@ -636,11 +636,11 @@ type QueryLimitSpec struct {
 // IngestionLimitSpec defines the limits applied at the ingestion path.
 type IngestionLimitSpec struct {
 	// DesiredRate defines the desired ingestion rate per second that LokiStack should
-	// target applyfing automatic stream sharding. Units Bytes.
+	// target applying automatic stream sharding. Unit MB.
```

Comment:
What we're actually using are `MiB`, should we mention this in the comment?

```suggestion
	// target applying automatic stream sharding. Unit MiB.
```